### PR TITLE
This is a PR

### PR DESCRIPTION
**Pull request by dmoreno-harveyRoad**
This is a PR message

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 This is a line edited with the online editor
 
 This is a line added in a different branch.
+
+More new lines
```
Comment (Galithil) [inline README.md]: This line sucks

File: README.md [hunk 2]
```diff
@@ -3,3 +3,5 @@
 This is a line edited with the online editor
 
 This is a line added in a different branch.
+
```
Comment (Galithil) [inline README.md]: This line is useless

File: README.md (hunk 1 above)
Comment (dmoreno-harveyRoad) [reply README.md]: you're dumb

File: README.md (hunk 2 above)
Comment (dmoreno-harveyRoad) [reply README.md]: I agree

Comment (dmoreno-harveyRoad) [reply README.md]: See commit https://github.com/dmoreno-harveyRoad/GitHub-Example/pull/2/commits/e9bd5ef4207cd5e2a77d8c4fd20dd22779c4e4f3

**Comment by Galithil** [comment]
Approved

**Pull request merged**